src/notebookRunGroups/util/cellMetadataHelpers.ts

```diff
@@ -26,6 +26,7 @@ export function updateCellRunGroupMetadata(cell: vscode.NotebookCell, newGroupVa
 
     // Perform our actual replace and edit
     const wsEdit = new vscode.WorkspaceEdit();
-    wsEdit.replaceNotebookCellMetadata(cell.notebook.uri, cell.index, newMetadata);
+    const notebookEdit = vscode.NotebookEdit.updateCellMetadata(cell.index, newMetadata);
```

Comment:
@rchiodo Thanks! You can always poke me for powertoy fixes as well. I don't mind fixing up mine at any time.